src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,35 +6,78 @@ cloud: true
 
-## Authorize the right network (Dedicated)
+## SQL Best Practices
 
-CockroachDB {{ site.data.products.dedicated }} requires you to authorize the networks that can access the cluster to prevent denial-of-service and brute force password attacks. During the application development phase, you might have authorized only your local machine’s network. To move into production, you need to authorize your application server’s network.
+To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
-To verify that you have authorized the application server's network, navigate to the [**Networking** page]({% link cockroachcloud/connect-to-your-cluster.md %}#authorize-your-network) on the CockroachDB {{ site.data.products.cloud }} Console and check if you see the application server network in the list of authorized networks. If you do not see the application server network in the list, [authorize the network]({% link cockroachcloud/connect-to-your-cluster.md %}#authorize-your-network).
+## Use a pool of persistent connections
 
-## Configure PCI ready features (Dedicated advanced)
+Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
+
+For guidance on sizing, validating, and using connection pools with CockroachDB, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
+
+## Keeping connections current
+
+After an application establishes a connection to CockroachDB {{ site.data.products.cloud }}, those connections can occasionally become invalid. This could be due to changes in the cluster topography, rolling [upgrades]({% link cockroachcloud/upgrade-policy.md %}) and restarts, network disruptions, or cloud infrastructure unavailability.
+
+Make sure connection validation and retry logic is used by your application. Validating and retrying connections is typically handled by the driver, framework, or the connection pool used by an application. For guidance on connection pool sizing, connection validation, and connection retry logic, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
```

Comment:
Do we have docs on connection retry logic specifically? We should definitely link that here -- it's a specific error code a lot of apps don't add to their retry logic.

Additionally, I know our connection pooling docs talk about setting max lifetime to 5min. Wonder if that's worth calling out explicitly here about why -- since a lot of customers don't and as a result their connections get severed by drain and then customers get sad when it adds latency / increases transaction errors

Reply:
Opened a tracking issue for error handling doc improvements: https://cockroachlabs.atlassian.net/browse/DOC-9237